\setcounter{page}{1}
\renewcommand{\thepage}{Data Management Plan - Page \arabic{page} of 2}
%\required{Supplementary Documentation}
\required{Data Management Plan}
%\begin{center}
%\emph{Maximum of 2 pages}
%\end{center}â€¢
% Maximum of 2 pages
%------------------------------

% This supplement should describe how the proposal will conform to NSF policy on
% the dissemination and sharing of research results and may include:
% 
% 1. The types of data, samples, physical collections, software, curriculum
%    materials, and other materials to be produced in the course of the project;
% 
% 2. The standards to be used for data and metadata format and content (where
%    existing standards are absent or deemed inadequate, this should be documented
%    along with any proposed solutions or remedies);
% 
% 3. Policies for access and sharing including provisions for appropriate
% protection   of privacy, confidentiality, security, intellectual property, or
% other rights or   requirements;
% 
% 4. Policies and provisions for re-use, re-distribution, and the production of
%    derivatives; and
% 
% 5. Plans for archiving data, samples, and other research products, and for
%    preservation of access to them.
% 
% A valid Data Management Plan may include only the statement that no detailed
% plan is needed, as long as the statement is accompanied by a clear
% justification. Proposers who feel that the plan cannot fit within the supplement
% limit of two pages may use part of the 15-page Project Description for
% additional data management information. Proposers are advised that the Data
% Management Plan may not be used to circumvent the 15-page Project Description
% limitation.

%(A-1) Sharing of Results and Management of Intellectual Property (maximum 3 pages): Describe the management of intellectual property rights related to the proposed project, including plans for sharing data, information, and materials resulting from the award. This plan must be specific about the nature of the results to be shared, the timing and means of release, and any constraints on release. The proposed plan must take into consideration the following conditions where applicable:
%
%Sequences resulting from high-throughput large-scale sequencing projects (low pass whole genome sequencing, BAC end sequencing, ESTs, full-length cDNA sequencing, etc.) must be released according to the currently accepted community standard (e.g. Bermuda/Ft. Lauderdale agreement) to public databases (GenBank if applicable), as soon as they are assembled and quality checked against a stated, pre-determined quality standard. "At publication" is not acceptable.
%Proposals that would develop genome-scale expression data through approaches such as microarrays or next-generation sequencing should meet community standards for these data (for example, Minimum Information About a Microarray Experiment or MIAME standards). The community databases (e.g. Gene Expression Omnibus) into which the data would be deposited, in addition to any project database(s) should be indicated.
%If the proposed project would produce genome-scale data sets generated using proteomics and/or metabolomics approaches, NSF encourages that they be made available as soon as their quality is checked to satisfy the specifications approved prior to funding. The timing of release should be stated clearly in the proposal. The community databases into which the data would be deposited, in addition to any project database(s) should be indicated.
%If the proposed project would produce community resources (biological materials, software, etc.), NSF encourages that they be made available as soon as their quality is checked to satisfy the specifications approved prior to funding. The timing of release should be stated clearly in the proposal. The resources produced must be available to all segments of the scientific community, including industry. A reasonable charge is permissible, but the fee structure must be outlined clearly in the proposal. If accessibility differs between industry and the academic community, the differences must be clearly spelled out. If a Material Transfer Agreement is required for release of project outcomes, the terms must be described in detail.
%When the project involves the use of proprietary data or materials from other sources, the data or materials resulting from NSF funded research must be readily available without any restrictions to the users of such data or materials (no reach-through rights). The terms of any usage agreements should be stated clearly in the proposal.
%Budgeting and planning for short-term and long-term distribution of the project outcomes must be described in the proposal. If a fee is to be charged for distribution of project outcomes, the details should be described clearly in the proposal. Letters of commitment should be provided from databases or stock centers that would distribute project outcomes, including an indication of what activities would be undertaken and funds needed for these activities (if any).
%In case of a multi-institutional proposal, the lead institution is responsible for coordinating and managing the intellectual property resulting from the PGRP award. Institutions participating in multi-institutional projects should formulate a coherent plan for the project prior to submission of the proposal.
%

\subsection*{Data Types}

This proposal will generate genotype and full-genome sequence data, phenotype data, analytical code, germplasm, and publications.

\subsection*{Data Archiving, Plan for Sharing, Public Access Policy}

\paragraph{Genotype and Sequence Data} 
All data will be made publicly available and stored online.  However, prior to public release, all data will be hosted locally.  Drs. Hufford and Ross-Ibarra will maintain a backup of all raw genotyping and sequencing data.  Dr. Hufford has access to 144Tb of free data storage through the College of Liberal Arts and Sciences at Iowa State and Dr. Ross-Ibarra maintains a DROBO distributed backup server (currently $>8$Tb of free space) which is robust to single disk failure. All sequence data (whole genome sequencing, and fastq files from genotyping by sequencing) will be submitted immediately upon completion of data quality control to the NCBI sequence read archive (SRA), along with passport information on each parent. A "hold until publication" embargo will be requested at the SRA. Just prior to publication, genotypes will be made publicly available via the Figshare website (\url{www.figshare.com}), a free public website allowing dissemination and archiving of large datasets. Data will be released in accordance with the Toronto agreement (2009. Nature 461:168-170. \url{www.nature.com/nature/journal/v461/n7261/full/461168a.html}) under the stipulation that no whole-genome analyses be performed until we have published our initial analyses.

\paragraph{Phenotype Data} 
Phenotypic data will be recorded digitally in the field using a high-throughput technique developed by Dr. Sherry Flint-Garcia at USDA and the University of Missouri.  Data will be uploaded at the end of each day into the FieldBook database developed by Dr. Flint-Garcia and immediately backed up at a remote location. Data will be grouped into projects, and each project is associated with a unique digital object identifier (DOI). Phenotypic data will then be uploaded to Figshare, along with appropriate metadata associated with other publications, links to germplasm, SRA experiments, Github code, etc.  Data on Figshare are publicly available and searchable.  We will submit data as soon as we complete quality control, but again with explicit stipulations as to the analyses that the data can be used for prior to our initial publication. All appropriate metadata including plant ID, data collector, field location, etc. will be associated with phenotype data deposited to Figshare. 

\paragraph{Analytical Software and Code} 
Analytical software and code from this project will be hosted on Github, a version-controlled public git repository.  Upon submission of papers all code will be made publicly available.  Drs. Hufford and Ross-Ibarra have already done this extensively (see \url{https://github.com/mbhufford} and \url{https://github.com/rossibarra} and \url{https://github.com/rilab}. Publication of all code will ensure reproducibility of all analyses conducted.  All data and code will be made publicly available via a creative commons CC by 2.0 license \url{http://creativecommons.org/licenses/by/2.0/} allowing free access to reuse, redistribute, and modify, requiring only citation of the license and the original source.

\paragraph{Germplasm} 
Sample accession data will be securely stored in MySQL servers hosted at Iowa State University and the University of California, Davis and backed up on a weekly basis offsite.  International agreements prohibit some of the maize and teosinte germplasm collected in Mexico and Guatemala from being stored and distributed by USDA.  We will, however, deposit small quantities of seed from all our collections with the CIMMYT germplasm bank in Mexico, which provides public access to seed.

\paragraph{Publications} 
All publications resulting from this project will be submitted to one or more preprint servers (e.g. arXiv, bioRxiv, PeerJ) such that they will be publicly available immediately upon submission of the paper for publication.



